single_kernel_mongo/config/statuses.py

```diff
@@ -379,71 +379,104 @@ def refreshing_needs_resume(
 class LdapStatuses(Enum):
     """Ldap Statuses."""
 
-    INVALID_LDAP_USER_MAPPING = StatusObject(
-        status="blocked",
-        message="Invalid LdapUserToDnMapping, please update your config.",
+    ACTIVE_IDLE = StatusObject(
+        status="active",
+        message="",
     )
-    INVALID_LDAP_QUERY_TEMPLATE = StatusObject(
-        status="blocked",
-        message="Invalid LDAP Query template, please update your config",
+    WAITING_FOR_DATA_AND_CERTS = StatusObject(
+        status="waiting",
+        message="Waiting for LDAP data and certificates.",
+        short_message="LDAP is not ready...",
     )
-    CONFIGURING_LDAP = StatusObject(
-        status="maintenance", message="Configuring LDAP", running="blocking"
+    WAITING_FOR_CERTS = StatusObject(
+        status="waiting",
+        message="Waiting for LDAP certificates...",
     )
-    INVALID_LDAP_REL_ON_SHARD = StatusObject(
-        status="blocked",
-        message="Cannot integrate LDAP with shard.",
+    WAITING_FOR_DATA = StatusObject(
+        status="waiting",
+        message="Waiting for LDAP data...",
     )
-    TLS_REQUIRED = StatusObject(
+    INVALID_USER_MAPPING = StatusObject(
         status="blocked",
-        message="TLS is mandatory for LDAP transport.",
+        message="Invalid LDAP to DN mapping config.",
```

Comment:
Maybe ? Or the name of the configuration option we use.
```suggestion
        message="Invalid LDAP User to DN mapping config.",
```